system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts

```diff
@@ -1,56 +1,60 @@
 ### Networking-DVS Alerts ###
 
-ALERT OpenstackDVSPolling
-  IF (min(rate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) by (cluster) *60) < 1
+ALERT OpenstackNovaDVSPolling
+  IF (min(rate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) by (region,kubernetes_pod_name) *60) < 1
```

Comment:
Region will be missing as long as #119 isn't fixed. Will try the alert relabeling tomorrow and remove by (region) if that works.